tests/features/common/kie-common.feature

```diff
@@ -124,4 +124,30 @@ Feature: RHPAM and RHDM common tests
       | AUTH_LDAP_LOGIN_MODULE | required  |
     Then file /opt/eap/standalone/configuration/standalone-openshift.xml should contain <login-module code="LdapExtended" flag="required">
 
+  Scenario: Check KIE_SERVER_JBPM_CLUSTER flag enabled
```

Comment:
You've used the wrong file, this test is only for RHDM and RHPAM KIE Server images, the kie-common will execute the tests on more images than just the KIE Server ones.

Please move to common/kie-kieserver-common.feature.

Reply:
Moved